app/src/main/java/org/p2p/wallet/striga/sms/onramp/StrigaOnRampSmsStorage.kt

```diff
@@ -0,0 +1,21 @@
+package org.p2p.wallet.striga.sms.onramp
+
+import org.p2p.wallet.striga.sms.StrigaSmsStorageContract
+import org.p2p.wallet.striga.user.StrigaStorageContract
+
+class StrigaOnRampSmsStorage(
+    private val storage: StrigaStorageContract
+) : StrigaSmsStorageContract {
+
+    override var exceededVerificationAttemptsMillis: Long
```

Comment:
you can do Duration instead of Long

or replace long with EpochSinceMillis

Reply:
replaced with EpochSinceMillis, because logic for Duration needs to be fully retested